streams/src/main/java/org/apache/kafka/streams/kstream/Branched.java

```diff
@@ -90,33 +104,43 @@ public static <K, V> Branched<K, V> withConsumer(final Consumer<KStream<K, V>> c
     /**
      * Create an instance of {@code Branched} with provided chain function and branch name suffix.
      *
-     * @param chain A function that will be applied to the branch. If the provided function returns
-     *              {@code null}, its result is ignored, otherwise it is added to the {@code Map} returned
-     *              by {@link BranchedKStream#defaultBranch()} or {@link BranchedKStream#noDefaultBranch()} (see
-     *              {@link BranchedKStream} description for details).
-     * @param name  the branch name suffix to be used. If {@code null}, a default branch name suffix will be generated
-     *              (see {@link BranchedKStream} description for details)
-     * @param <K>   key type
-     * @param <V>   value type
+     * @param chain
+     *        A function that will be applied to the branch. If the provided function returns
+     *        {@code null}, its result is ignored, otherwise it is added to the {@code Map} returned
+     *        by {@link BranchedKStream#defaultBranch()} or {@link BranchedKStream#noDefaultBranch()} (see
+     *        {@link BranchedKStream} description for details).
+     * @param name
+     *        the branch name suffix to be used. If {@code null}, a default branch name suffix will be generated
+     *        (see {@link BranchedKStream} description for details)
+     *
+     * @param <K> key type
+     * @param <V> value type
+     *
      * @return a new instance of {@code Branched}
      */
     public static <K, V> Branched<K, V> withFunction(
-            final Function<? super KStream<K, V>, ? extends KStream<K, V>> chain, final String name) {
+        final Function<? super KStream<K, V>, ? extends KStream<K, V>> chain,
```

Comment:
Ditto here, I feel these newlines are not very necessary?

Reply:
Not necessary. It's my personal preferred style. Happy to change it back. Please let me know.

Reply:
I just personally feel a newline with just an extra param feels not very elegant. Again just my own thoughts, feel free to ignore.